### PR TITLE
Issues/nameclash explicit interface renamed methods

### PR DESCRIPTION
**Pull request by StealthyDeveloper**
Link to issue(s) this covers

### Problem
#749

### Solution
When interface members are renamed, emit explicit interface implementations. 
Treat casing differences as a rename.
Added support for multiple interface implements.

Most important code:
CommonConversions.WithDeclarationName
DeclarationNodeVisitor.VisitPropertyStatement
DeclarationNodeVisitor.VisitMethodStatement 

Added multiple tests in MemberTests.

Note: since renaming of interface members no longer is the responsibility of CommonConversions.WithDeclarationName, there's no longer a need to check the convertedassemblies (#731)

### CODE REVIEW

File: CodeConverter/CSharp/CommonConversions.cs [hunk 1]
```diff
@@ -428,15 +473,15 @@ private IEnumerable<CSSyntaxKind> CsSyntaxAccessibilityKind(Accessibility declar
...
+                    return new[] {CSSyntaxKind.ProtectedKeyword};
                 case Accessibility.Internal:
-                    return new[] { CSSyntaxKind.InternalKeyword };
+                    return new[] {CSSyntaxKind.InternalKeyword};
                 case Accessibility.ProtectedOrInternal:
-                    return new[] { CSSyntaxKind.ProtectedKeyword, CSSyntaxKind.InternalKeyword };
+                    return new[] {CSSyntaxKind.ProtectedKeyword, CSSyntaxKind.InternalKeyword};
                 case Accessibility.Public:
-                    return new[] { CSSyntaxKind.PublicKeyword };
+                    return new[] {CSSyntaxKind.PublicKeyword};
```
Comment (GrahamTheCoder) [inline CodeConverter/CSharp/CommonConversions.cs:484]: I realize the repo is pretty much half and half on this convention. If you want to standardize it, please create a separate PR for that

File: CodeConverter/CSharp/CommonConversions.cs [hunk 2]
```diff
@@ -403,7 +446,8 @@ private static bool ContextHasIdenticalDefaults(TokenContext context, TokenConte
             var remainingModifiers = modifiers.ToList();
             if (declaredAccessibility != Accessibility.NotApplicable) {
                 remainingModifiers = remainingModifiers.Where(m => !m.IsVbVisibility(false, false)).ToList();
-                foreach (var visibilitySyntaxKind in CsSyntaxAccessibilityKindForContext(declaredAccessibility, context)) {
+                foreach (var visibilitySyntaxKind in
+                    CsSyntaxAccessibilityKindForContext(declaredAccessibility, context)) {
```
Comment (GrahamTheCoder) [inline CodeConverter/CSharp/CommonConversions.cs:450]: I'd be happy with a PR that wraps everything to 200 characters. 120 character wrapping randomly mixed with code changes is hard to review even with your pointers in the description

Comment (GrahamTheCoder) [reply CodeConverter/CSharp/CommonConversions.cs:450]: Could you undo these wrapping changes so I can review the PR properly please? (At least ones that you can easily revert where no functionality changed)

Comment (StealthyDeveloper) [reply CodeConverter/CSharp/CommonConversions.cs:450]: Yes I will undo them tomorrow. Btw is the standard in this repo wrap at 200? I could edit the .editorconfig then.

Comment (GrahamTheCoder) [reply CodeConverter/CSharp/CommonConversions.cs:450]: There isn't currently a standard in the repo, but yes let's set that as the standard - feel free to set it to that in the .editorconfig.
Hopefully that leaves auto-wrapping as a rare last resort and most of the times humans will format things in the way they feel is most readable.

Comment (StealthyDeveloper) [reply CodeConverter/CSharp/CommonConversions.cs]: i reverted the formatting changes and set the wrap limit at 200 chars.

File: CodeConverter/CSharp/CommonConversions.cs (hunk 1 above)
Comment (StealthyDeveloper) [reply CodeConverter/CSharp/CommonConversions.cs]: I also reverted this with the other formatting changes in this file (might as well).

**Comment by GrahamTheCoder** [comment]
Looks great, nice one!

**Pull request merged**